src/tools/rbd/action/BenchWrite.cc

```diff
@@ -262,6 +262,11 @@ int execute(const po::variables_map &vm) {
   } else {
     bench_io_size = 4096;
   }
+  if ((bench_io_size > 100*1024*1024) || (bench_io_size < 1))
```

Comment:
@x11562 I wasn't able to reproduce on a trusty VM.  Do you have memory restricted via cgroups or ulimits?